front/app/containers/Admin/reporting/containers/ReportBuilder/index.tsx

```diff
@@ -82,22 +123,43 @@ const ReportBuilder = ({ report, reportLayout }: Props) => {
     Object.values(contentBuilderErrors).filter((node) => node.hasError).length >
     0;
 
+  const handleSetSaved = () => {
```

Comment:
Just to make it clearer that `saved` is not set to `false` in the child component.